### PR TITLE
I18n for use location

### LINKED ISSUE #38: Internationalize 'use my location' control
Followup from #33

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fixes #38 

Uh, better traanslations welcome!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h2 data-i18n="app.title"></h2>
                             <input class='col8' id='address' placeholder='' data-i18n="[placeholder]step1.addr"/>
                             <button class='col4' id='findme' data-i18n="step1.button"></button>
                             <br />
-                            Or <a id="use_my_location" href="#" onclick="return false">use my current location</a>.
+                            <a id="use_my_location" data-i18n="step1.use_location" href="#" onclick="return false">use my current location</a>.
```
Comment (alexandre-mbm) [inline index.html:48]: I like this "Or". I do not like the lower case expression.

Comment (CloCkWeRX) [reply index.html:48]: Yeah, I did this to make it easier to translate without having to embed the links in the translated string. I might have a look at bringing it back if we add more controls down there, like 'import from facebook' or similar.

Comment (alexandre-mbm) [reply index.html:48]: I am thinking a button like [this](https://cdn3.iconfinder.com/data/icons/glypho-travel/64/gps-position-target-512.png) between the field and the current button. The translation would be for a tooltip, without the "or".

Comment (CloCkWeRX) [reply index.html:48]: I did see the italian version ended up up using https://github.com/domoritz/leaflet-locatecontrol after this PR, I might revisit it a bit later to move us to that if the 'from facebook' or similar extra tools don't make it in.

**Pull request merged**